src/python/zquantum/qaoa/ansatzes/warm_start_ansatz.py

```diff
@@ -0,0 +1,87 @@
+from zquantum.core.interfaces.ansatz import Ansatz, ansatz_property
+from zquantum.core.circuit import Circuit, Qubit, create_layer_of_gates
+from zquantum.core.evolution import time_evolution
+from zquantum.core.openfermion import qubitop_to_pyquilpauli, change_operator_type
+
+from openfermion import QubitOperator, IsingOperator
+from openfermion.utils import count_qubits
+from typing import Union, Optional
+import numpy as np
+import sympy
+from overrides import overrides
+
+
+class WarmStartQAOAAnsatz(Ansatz):
+
+    supports_parametrized_circuits = True
+    cost_hamiltonian = ansatz_property("cost_hamiltonian")
+    thetas = ansatz_property("thetas")
+
+    def __init__(
+        self,
+        number_of_layers: int,
+        cost_hamiltonian: Union[QubitOperator, IsingOperator],
+        thetas: np.ndarray,
```

Comment:
`thetas` input is not described

Reply:
Fixed in https://github.com/zapatacomputing/z-quantum-qaoa/pull/31/commits/0bdf13c508c63ae5f8f26ed819fb7b57b4ba9110